dockstore-common/src/main/java/io/dockstore/common/yaml/YamlNotebook.java

```diff
@@ -100,6 +103,22 @@ public void setPath(final String path) {
         this.path = path;
     }
 
+    /**
+     * Get the kernel image reference for the notebook.
```

Comment:
Would be nice to have some examples to indicate why this could not be stored in `Image`. perhaps with an enum?